CONTRIBUTING.md

```diff
@@ -1,4 +1,49 @@
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
+
 ## Contributing to this Project
 
-OpenSearch is a community project that is built and maintained by people just like **you**.
+    OpenSearch is a community project that is built and maintained by people just like **you**.
 [This document](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md) explains how you can contribute to this and related projects.
+The brief summary below outlines contribution guidelines specific to the OpenSearch-Security Repo.
+
+## Triaging
+
+OpenSearch-Security seeks to promote an inclusive and engaged community of contributors. In order to facilitate this, weekly triage meetings are open-to-all and attendance is encouraged for anyone who hopes to contribute, discuss an issue, or learn more about the project.
```

Comment:
Who is `OpenSearch-Security`?  There are a few other teams/groups that could be mistaken by just "the contributors of the OpenSearch-Security repo". I would either say "OpenSearch" (for which the statement applies as well, I hope _all_ repos do seek to promote an inclusive and engaged community of contributors) or replace by something like that "the contributors of this repo".

Reply:
The problem is that I do not know that it is appropriate to speak for all the teams. Later, I also mention who can people contact and saying OpenSearch is not really fair since they did not sign up for it and saying "contributor to this repo" does not work either since there are many contributors who have not agreed to this. I can put maintainers however.